SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -267,7 +405,7 @@ Interest UnrealServerInterestFactory::CreateInterest(AActor* InActor, const FCla
 	{
 		if (GetDefault<USpatialGDKSettings>()->bUseClientEntityInterestQueries)
 		{
-			AddClientInterestEntityIdQuery(ResultInterest, InActor);
+			// CreateClientInterestDiff(InActor);
```

Comment:
Can we return here? Otherwise we're executing the old logic to populate the Interest struct?

Reply:
(and please just add a comment say we create diff via a different path)

Reply:
still need to create interest for server workers